src/Microsoft.AspNetCore.Razor.Sdk/build/netstandard2.0/Razor.Sdk.CurrentVersion.targets

```diff
@@ -161,6 +161,46 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="RazorCompile" DependsOnTargets="$(RazorCompileDependsOn)">
   </Target>
 
+  <!--
+    Generates assembly attributes in support for Razor runtime code generation. This is a set of standard
+    metadata attributes (defined in Microsoft.AspNetCore.Razor.Runtime) that capture the build-time
+    Razor configuration of an application to be used at runtime.
+    
+    This allows the project file to act as the source of truth for the applicable Razor configuration regardless 
+    of how Razor is used.
+
+    The SDK expects configurations that use runtime compilation to set $(GenerateRazorAssemblyInfo) to true,
+    it will be unset by default.
+  -->
+  <Target 
+    Name="RazorGetAssemblyAttributes" 
+    AfterTargets="GetAssemblyAttributes"
+    Condition="'$(GenerateRazorAssemblyInfo)'=='true' and '$(RazorDefaultConfiguration)'!=''">
+
+    <ItemGroup>
+      <_ResolvedRazorConfiguration Include="@(RazorConfiguration)" Condition="'%(RazorConfiguration.Identity)'=='$(RazorDefaultConfiguration)'" />
+      <_ResolvedRazorExtensionName Include="%(_ResolvedRazorConfiguration.Extensions)"/>
+    </ItemGroup>
+
+    <FindInList List="@(RazorExtension)" ItemSpecToFind="%(_ResolvedRazorExtensionName.Identity)">
```

Comment:
Inline it - `<FindInList List="@(RazorExtension)" ItemSpecToFind="%(_ResolvedRazorConfiguration.Identity)">`?

Reply:
This is intended. `_ResolvedRazorConfiguration` is a list that gets created from a single string.